tests/e2e/e2e_suite_test.go

```diff
@@ -15,11 +15,18 @@ var ctx context.Context
 var token string
 var clusterID string
 
+func GetToken() string {
+	return token
+}
+
+func GetClusterID() string {
+	return clusterID
+}
+
 func TestRHCSProvider(t *testing.T) {
-	RegisterFailHandler(Fail)
 	token = os.Getenv(CON.TokenENVName)
 	clusterID = CI.PrepareRHCSClusterByProfileENV()
 	ctx = context.Background()
-	RunSpecs(t, "RHCS Provider Test")
-
+	RegisterFailHandler(Fail)
```

Comment:
Please RegisterFailHandler at the beginning , otherwise unexpected error cannot be caught